charts/mailhog/templates/deployment.yaml

```diff
@@ -30,6 +30,10 @@ spec:
       securityContext:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.image.pullSecrets }}
```

Comment:
Don't group it under `image`.
```suggestion
      {{- with .Values.imagePullSecrets }}
```

Reply:
Done. At this point, after configuring this setting in ~20 different helm charts I'm confident to say that I've seen almost every possible approach to setting this value :D

Reply:
Yes, agreed. But this way is consistent with how it is in the manifest.